slick/src/sphinx/developers.rst

```diff
@@ -0,0 +1,155 @@
+For Slick developers
+======================
+
+`slick at github <https://github.com/slick/slick>`_
+
+`waffle board <https://waffle.io/slick/slick>`_
+
+`slick-extensions at github <https://github.com/typesafe-hub/slick-extensions>`_
+
+
+Clone source code
+------------------
+``git clone https://github.com/slick/slick.git``
+
+Run tests
+----------
+all
+___
+``sbt> test``
+
+selected
+________
+
+``sbt> test-only -- *MainTest*`` or some other wildcard using a partial test class or method name
+
+enable output
+______________
+``sbt> test-only -- +q``
+
+enable Slick Scala to SQL compiler debug log
+______________________________________________
+
+``sbt> test-only -- -Dlog.qcomp=debug -Dscala.slick.ansiDump=true``
```

Comment:
`scala.slick.ansiDump` is now `slick.ansiDump` (pretty sure about that, but not tried it).  It looks like it's `slick.ansiDump` from the description in the Configuration section of this manual.